web/controllers/friends_controller.ex

```diff
@@ -16,6 +16,18 @@ defmodule Entice.Web.FriendsController do
       friends: friends}
   end
 
+  @doc "Returns a friend's online status and current playing character name."
+  def status(conn, _params) do
+    id = get_session(conn, :client_id)
+    account_id = conn.params["account_id"]
+
+    result = case Client.get_status(account_id) do
+      {:error, :no_matching_account} -> error(%{message: "There is no account with that id."})
+      {:ok, status, name} -> ok(%{message: "Friend info recovery successful.", online_status: status, character_name: name})
+    end
+
+    conn |> json result
+  end
 
```

Comment:
i think we should also be able to reply to batch requests, i.e. if the client sends several account_ids at the same time. otherwise loads of spam is to be expected...
we might even find it too much for what we want to achieve in the future and serve this by means of channels, but thats another story